src/components/TreeVisualizer.jsx

```
@@ -156,66 +157,221 @@ const bstInsert = (root, value) => {
   
-  return { tree: newRoot, results };
+  return { tree: finalTree, results };
+};
+
+// Helper function to remove highlights from a tree
+const removeHighlights = (node) => {
+  if (!node) return;
+  
+  // Remove highlight attributes
+  delete node.highlight;
+  delete node.pathHighlight;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)